doc/about/user-guidelines.md

```diff
@@ -0,0 +1,67 @@
+# `mybinder.org` Usage Guidelines
+
+This page details some guidelines and policies that we follow at ``mybinder.org``.
+
+:::{admonition,warning} When in doubt, open an issue
+If you're not sure whether your use-case is reasonable for `mybinder.org`, don't
+hesitate to [open an issue](https://github.com/jupyterhub/mybinder.org-deploy/issues/new?labels=impact&template=request_resources.md)
+to ask if your planned usage is OK.
+:::
+
+## Temporary Banning
+
+Temporary banning means that `mybinder.org` will stop building / serving Binder
+sessions for a given repository. This usually happens because of some
+undesired behavior with the repository. Some examples of behavior that could
+result in temporary banning include:
+
+* A large, unexpected spike in traffic that persists over time.
+* Sessions that routinely use a large amount of CPU.
+* Sessions that attempt to perform a lot of outgoing networking traffic.
+
+The `mybinder.org` team uses the following workflow in discussing / deciding
+when to temporarily ban a repository:
+
+* An issue is created to document why a repo (or group of repos) is being banned
+* A PR that implements the ban is opened
+* A "bans" tag notes any PRs that created a ban
+* The issue stays open until the ban is lifted
+
+If you are temporarily banned, contact us on the
+[Gitter Channel](https://gitter.im/jupyterhub/binder) or
+[Open an Issue](https://github.com/jupyterhub/mybinder.org-deploy/issues) to discuss
+how to un-ban the repository.
+
+## Running events with `mybinder.org`
+
+If you'd like to run an event with `mybinder.org` and expect more than 50 users
+to join at once, please [request an increase in quota for your repository](https://github.com/jupyterhub/mybinder.org-deploy/issues/new?labels=impact&template=request_resources.md).
+
+The Binder Team occasionally increases the number of sessions that can run for a single repository if it is being used for purposes that align with the mission of the Binder Project. However, there is no guarantee that this will be done for your event.
+
+## Maximum Concurrent Users for a Repository
+
+We don't want a single repository to dominate all of the traffic to Binder, so
+we've set a maximum limit of concurrent user sessions that point to the same
+Binder link. **The maximum number of simultaneous users for a given repo is 100**.
+If you think you have a *really* good reason for why this number should be
+higher, please [Open an Issue](https://github.com/jupyterhub/mybinder.org-deploy/issues).
+to discuss with the community!
+
+## Maximum Concurrent Users for a Service
```

Comment:
```suggestion
## Using mybinder.org as backend for your service

Please do not use mybinder.org as backend for your for-profit service or product.

The cloud resources consumed by mybinder.org are funded through donations or grants. Because
of this we can not subsidise for-profit products or services by offering free cloud resources to them.
This means we will prevent you from using mybinder.org as a service for your product.

Operating mybinder.org relies on the good will of volunteers and organisations to obtain the
resources required to do so. This is easier if we are not seen as supporting for-profit services
that are not a net contributor to the service.
```

What do you think of this? I thought that the new title was easier to find for someone thinking about providing a service based on mybinder.org. I think this also has a simpler message ("don't do it") instead of going into details of when/if/how you might run into trouble. I think the general principle is "don't do it" which is what readers should care about. This is because we might fix the load issues related to builds which might make you think it is now Ok because we said "builds cause trouble so we stop you". Whereas in reality it remains not Ok because the general rule is "don't do it'.

(this suggestion should probably replace all of this FAQ item)

Reply:
I think this is a reasonable explanation - I guess my main question is how we'd differentiate between "spacy-style backends" and "jovian.ml-style backends"? Is the decision question "do you directly make profit from running the service that Binder is powering for you?"